DZ8/spravka.py

```diff
@@ -0,0 +1,64 @@
+import json
+import generate_person
+
+data = {}
+
+for i in range(1,6):
+    data[str(i)] = generate_person.person_data()
```

Comment:
```suggestion
    data[i] = generate_person.person_data()
```
Но обязательно делать строкой.